Arrowgene.Ddon.GameServer/Quests/QuestStateManager.cs

```diff
@@ -950,7 +950,7 @@ public PacketQueue HandleEnemyHuntRequests(Enemy enemy, DbConnection? connection
 
                         S2CQuestQuestProgressWorkSaveNtc ntc = new()
                         {
-                            QuestScheduleId = quest.Key,
+                            QuestScheduleId = questScheduleId,
```

Comment:
Oops

Reply:
Key and schedule id are two different values. I could never figure out what key value was based on and it is left as zero generally.